packages/web-frontend/src/screens/mobile/RootScreen/index.js

```diff
@@ -26,21 +34,66 @@ const EntityName = styled.p`
   color: white;
 `;
 
-const RootScreen = ({ orgUnit, isLoading, isUserLoggedIn, enlargedDialogIsVisible }) => (
-  <StyleRoot>
-    <EnvBanner />
-    <LoadingScreen isLoading={isLoading} />
-    <div>
-      <HeaderBar />
-      <EntityName>{orgUnit.name}</EntityName>
-      <SearchBar />
-      <Dashboard />
-      {enlargedDialogIsVisible ? <EnlargedDialog /> : null}
-      <Footer />
-      {isUserLoggedIn && <OverlayDiv />}
-    </div>
-  </StyleRoot>
-);
+const StyledTabs = styled(Tabs)`
+  .MuiTabs-scroller.MuiTabs-indicator {
+    display: none;
+  }
+`;
+
+const StyledTab = styled(Tab)`
+  flex: 1;
+  color: white;
+  background-color: ${({ selected }) => (selected ? TUPAIA_ORANGE : 'transparent')};
+  border: ${({ selected }) => (selected ? 'none' : `1px ${TUPAIA_ORANGE} solid`)};
+  text-transform: none;
+  opacity: 1;
+  max-width: none;
+`;
+
+const ComingSoon = styled.p`
+  color: white;
+  font-size: 32px;
+  text-align: center;
+  width: 100%;
+`;
+
+const RootScreen = ({
+  orgUnit,
+  isLoading,
+  isUserLoggedIn,
+  enlargedDialogIsVisible,
+  selectedTab,
+  setSelectedTab,
+}) => {
+  const handleChangeSelectedTab = (event, newValue) => setSelectedTab(newValue);
+
+  return (
+    <StyleRoot>
+      <EnvBanner />
+      <LoadingScreen isLoading={isLoading} />
+      <div>
+        <HeaderBar />
+        <EntityName>{orgUnit.name}</EntityName>
+        <StyledTabs
+          value={selectedTab}
+          onChange={handleChangeSelectedTab}
+          TabIndicatorProps={{
+            style: { display: 'none' },
+          }}
+        >
+          <StyledTab label="Dashboard" value="dashboard" disableRipple />
+          <StyledTab label="Map" value="map" disableRipple />
+        </StyledTabs>
+        <SearchBar />
+        {selectedTab === 'dashboard' && <Dashboard />}
+        {selectedTab === 'map' && <ComingSoon>Coming soon</ComingSoon>}
```

Comment:
Love the use of this 'Coming Soon' place holder 👍 Out of curiosity are you planning on releasing with that at any point, or just using it internally on the RN-373 branch?

Reply:
Yeah that's just internal for now - it's added in a later PR :)